.github/workflows/android-arm64_nightly.yaml

```diff
@@ -0,0 +1,14 @@
+name: android-arm64_nightly
+
+on:
+  schedule:
+    - cron: '0 14 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: ./.github/workflows/main.yaml
```

Comment:
I had assumed that nightly builders would _not_ be performing builds themselves, instead triggering other existing builders and using their outputs (similar to i.e. playstation nightlies today). Is that not the case?

Reply:
Chatted with Andrew offline about this.  Concepts of workflow, jobs, and steps in GitHub CI are different from Buildbot.  GitHub Actions not very good at having "builders" triggering other "builders", although there are ways to do that.